__tests__/__integration__/imperative/__tests__/__integration__/cli/config/convert-profiles/__scripts__/create_profiles.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+imperative-test-cli profiles create secured test --info hello --secret world
```

Comment:
Curious if the scripts could be reused or maybe more generalized?
That way we don't have to keep adding new script in case new tests have to be written in the future 😋 
(Kind of the same as other scripts we have used for other test-cli commands)

Otherwise, I'd suggest making the script-file name a little more specific to the test it's being used for 😅

Reply:
Will rename the scripts to make them more specific 🙂